agenta-cli/agenta/cli/variant_commands.py

```diff
@@ -259,11 +259,20 @@ def remove_variant_cli(variant_name: str, app_folder: str):
 @click.option('--file_name', default='app.py', help="The name of the file to run")
 def serve_cli(app_folder: str, file_name: str):
     """Adds a variant to the web ui and serves the api locally."""
-    config_check(app_folder)
-    host = get_host(app_folder)
-    variant_name = add_variant(variant_name='', app_folder=app_folder, file_name=file_name, host=host)
-    if variant_name:  # otherwise we either failed or we were doing an update and we don't need to manually start the variant!!
-        start_variant(variant_name=variant_name, app_folder=app_folder, host=host)
+    try:
+        config_check(app_folder)
+        host = get_host(app_folder)
+        variant_name = add_variant(variant_name='', app_folder=app_folder, file_name=file_name, host=host)
+        if variant_name:  # otherwise we either failed or we were doing an update and we don't need to manually start the variant!!
+            start_variant(variant_name=variant_name, app_folder=app_folder, host=host)
+    except Exception as e:
+        if "HTTPConnectionPool" and "Connection refused" in str(e):
```

Comment:
This is not the right way to do this!!
The right way is to import the exception that you want to handle in this case maybe `requests.exceptions.ConnectionError` and handle that Exception 

```suggestion
        except ConnectionError as e:
             somethingsomething
        except Exception as e:
             something else
```

Reply:
Thanks for your feedback! I will update the code to handle the error properly!.